### DISCUSSION TITLE
2.0版本什么时候出正式版啊？

### DISCUSSION THREAD

**Discussion by FCSDFSDVXDFG** [the1812/Bilibili-Evolved#2317] [综合讨论 / General]
真·简化主页 模块做得怎么样了啊，以及夜间模式的优化现在是不是停更了

**Comment by the1812** [comment]
还有一轮 TP, 然后上 Preview, 最后上正式, 大概 10 月底?

> 真·简化主页 模块做得怎么样了啊

那个不是说等正式版做完再开工吗(

> 以及夜间模式的优化现在是不是停更了

夜间模式因为 @Coulomb-G 暂离了, 更新速度现在是极慢的

**Comment by Geeyun-JY3** [comment]
翻翻 issue，能看到挺多感知挺明显的 bug 的（而且最近一直在冒出新 bug），光看这些 bug 就感觉至少还需要一段时间（
